README.md

```diff
@@ -105,3 +105,8 @@ As a minimum, any modules recommended as part of this reference architecture sho
 ## Contributing
 
 To Contribute to this project, please see the [Contributing Guide](./CONTRIBUTING.md).
+
+## Contributors
+
+* Michael Dawson - Node.js lead for Red Hat and IBM
+* Lucas Holmquist - Sr. Software Engineer - Red Hat
```

Comment:
Add:
 * Jim Lindeman - Sr. Software Engineer - IBM

Reply:
```suggestion
* Dominic Harries - Dev Lead at IBM Garage for Cloud
```

Reply:
Add:

* Dominic Harries - Dev Lead at IBM Garage for Cloud